baselines/bigquery-storage/package.json

```diff
@@ -24,7 +24,7 @@
     "test": "c8 mocha build/test"
   },
   "dependencies": {
-    "google-gax": "^2.2.0"
+    "google-gax": "next"
```

Comment:
Shouldn't that be a version number?

Reply:
yes, I'm waiting for the new version to release. Right now it's still in `next` version